src/utils.js

```diff
@@ -51,6 +51,7 @@ const getDaysSince = (current, previous) => {
 }
 
 const collectCommandAnalytics = async (commandName, subCommandName) => {
+    return; // SHUT UP AND STOP BOTHERING ME WITH EVENTS.JS OH MY GOD JUST SHUT UP
```

Comment:
Haha, you can just create an event.js file with an empty JSON array and it will work fine. Sorry for that, I should update it so that it creates file on its own, instead of crashing if there isn't one.
```suggestion
 
```

Reply:
Oh no I didn't meant to commit this, I forgot to uncheck this lol, I did what you said before but sometimes the file kept getting cleared so I was getting mad haha